.github/workflows/changed_file.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
Would it be better if this file is named as `changed_example` instead of `changed_file.py`?

Reply:
Meanwhile, follow the convention, you should put any related scripts inside `.github/workflows/scripts`.

Reply:
As the `scripts` folder contains other files, name your scripts with a common prefix or postfix such that people know these scripts are all related to examples.